R/Report_KRI.R

```diff
@@ -13,19 +13,20 @@
 
 #' @keywords KRI report
 #' @export
-#' 
+#'
 
 Report_KRI <- function(
   lCharts = NULL,
   dfSummary = NULL,
   dfStudy = NULL,
   dfSite = NULL,
   dfMetrics = NULL,
-  strOutpath = NULL
+  strOutpath = NULL,
+  strType = NULL
```

Comment:
I don't think this is needed. See the `group` derivation in `Report_Setup()` instead.

Reply:
@jwildfire , this should be complete now on the most recent commit.